src/containers/_RegistrationContainer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Registration } from '../components'
+import { useHistory } from 'react-router-dom'
+
+const RegistrationContainer = () => {
+
+	const history = useHistory()
+
+	const onContinue = () => {
+		history.push('/signup/regform')
+	}
+
+	return (
+		<Registration>
+
+			<Registration.Inner>
+				<Registration.Image src="/images/registration.png"/>
```

Comment:
add an `alt` attribute for your images sire

Reply:
Just fixed it